### PR TITLE
Polishing Kubernetes section view in Preferences

### PR DESCRIPTION
**Pull request by aleksfront**
* Using same components for iterable elements inside Kubesync and Helm Charts sections
* Moving some of the styles to CSS Modules
* Adding empty states for Kubesync and Helm Charts sections

kubernetes page

syncs empty state
helm charts empty state

### CODE REVIEW

**Review by Nokel81 — APPROVED** [review]

File: src/renderer/components/layout/setting-layout.scss [hunk 1]
```diff
@@ -228,7 +228,7 @@
       margin-top: 0;
       margin-bottom: 8px;
       padding-bottom: 0;
-      font-size: 12px;
```
Comment (Nokel81) [inline src/renderer/components/layout/setting-layout.scss:231]: Why this change?

Comment (aleksfront) [reply src/renderer/components/layout/setting-layout.scss:231]: For better readability. I found this font too small to read comfortably.

Comment (Nokel81) [reply src/renderer/components/layout/setting-layout.scss:231]: Sounds good
